README.md

```diff
@@ -1,4 +1,62 @@
-MolSim
+MolSim2024 - Group D
 ===
 
-The Molecular Dynamics teaching code.
+## Prerequisites
+
+Developed and tested with these versions, other versions might work
+
+- clang 17.0.6
+- lldb 17.0.6
+- cmake 3.27.7
+- ninja 1.11.1
+
+These environment variables are assumed to be always set
+
+```shell
+export RELEASE_BUILD_DIR=build-release
+export DEBUG_BUILD_DIR=build-debug
+
+export BUILD_DIR=RELEASE_BUILD_DIR # or DEBUG_BUILD_DIR if debugging
+```
```

Comment:
This is a bit weird. Just looking at the README I'd assume that you make use of them  somewhere, especially since you use `export` but it seems you only use them to explain your build process?

Just to be clear: I think it's GOOD that you don't use them further and let CMake do its thing. But maybe clarify this better ;)